storage/storage-interface/src/lib.rs

```diff
@@ -477,6 +489,25 @@ pub trait DbReader<PS: ProtocolSpec>: Send + Sync {
             ledger_version,
         )
     }
+
+    /// Returns total number of accounts at given version.
+    fn get_account_count(&self, version: Version) -> Result<usize> {
+        unimplemented!()
+    }
+
+    /// Get a chunk of account data, addressed by the index of the account.
+    fn get_account_chunk_with_proof(
+        &self,
+        version: Version,
+        start_idx: usize,
+        chunk_size: usize,
+    ) -> Result<AccountStatesChunkWithProof> {
+        unimplemented!()
+    }
+
+    fn get_state_prune_window(&self) -> Option<usize> {
```

Comment:
nit: can we also add a comment to the top of this method? It seems like everything else is documented 😄